src/e-init.js

```diff
@@ -16,15 +16,19 @@ function getVSReleaseLine() {
   return childProcess.execFileSync(exec, args, opts).trim();
 }
 
-function createConfig(name, options) {
+function createConfig(options) {
   const root = resolvePath(options.root);
   const homedir = os.homedir();
 
   // build the `gn gen` args
-  const gn_args = [
-    `import("//electron/build/args/${options.import}.gn")`,
-    `cc_wrapper="${sccache.exec(root)}"`,
-  ];
+  const gn_args = [`import("//electron/build/args/${options.import}.gn")`];
+
+  if (options.useGoma) {
+    gn_args.push('import("//electron/build/args/goma.gn")');
```

Comment:
We should only do this if this file actually exists.  `build-tools` still needs to work for < master